google/cloud/apigee/registry/applications/v1alpha1/registry_manifest.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package google.cloud.apigee.registry.applications.v1alpha1;
+
+option java_package = "com.google.cloud.apigee.registry.applications.v1alpha1";
+option java_multiple_files = true;
+option java_outer_classname = "RegistryManifestProto";
+option go_package = "github.com/apigee/registry/rpc;rpc";
+
+// Manifest represents the dependency relation between resources, which 
+// is used by the controller to keep the resources up-to-date.
+message Manifest {
+  // List of dependency relations.
+  repeated Resource resources = 1 [
+		(google.api.field_behavior) = REQUIRED
```

Comment:
There are two different descriptions here in the comment and field name/type: "dependency relations" and "resources". I think we'd get more conceptual clarity if they were consistent. "Resource" seems too generic, because not all resources have dependencies. "Dependency" seems too specific because any of the things listed here might have multiple dependencies. "GeneratedResource" is more descriptive but verbose. I like "Product" but think that it might be dismissed for the same reason that "Target" was. I've also seen "Result" considered, but that seems unclear (a result of what?).

From this, I think "generated resource" might be best - it's verbose but descriptive and accurate.
```
  // List of generated resources.
  repeated GeneratedResource generated_resources = 1;
```

Reply:
I like `GeneratedResource`.

The only reason I was avoiding multiple words in the name is because we have to reference it in the `update_action` or dependency `pattern` strings. I have used `$resource` and `$dependency` in the examples below. We can revisit if `GeneratedResource` and `$resource` feel a little disconnected.